### ISSUE TITLE
[Woo POS] Make Processing and Complete states separate screen (PaymentsView?) to prevent strange reader-not-connected error

### ISSUE THREAD

**Issue by joshheald** [woocommerce/woocommerce-ios#13876]
## Description
> When the Reader is disconnected (by turning off Bluetooth on the iPad) while waiting for a card to be tapped, the reader error appears full screen. A correct behavior would be to appear in split screen, the same way as before initiating the payment.
> 
> ![image](https://github.com/user-attachments/assets/070f0497-cd84-4bbe-94a0-e96a774315ee)
> 
> When the Reader is disconnected (by turning off Bluetooth on the iPad) while processing payment or payment success we can get into weird states. Although these states seem functional and after re-connection, we transition to a payment state, we could make improvements for even more solid error handling.
> ![image](https://github.com/user-attachments/assets/86ccf9a8-6625-4287-87e5-f6dc56d6e935)
> ![image](https://github.com/user-attachments/assets/56f59ca6-bc30-437d-b6c4-307ebeb5e85e)
> – p91TBi-c11-p2#comment-12991

When the reader gets disconnected, we could move back to the checkout stage.

However, if we do that from the Order Completed screen, or even the processing screen we risk people using a _paid_ order a second time.

If they do that (change the contents and check out again) the order will succeed without a card being tapped.

This means we'd be risking merchants giving products away for free, if they got confused by this.

We should fix this in our architecture, by taking the post-card tap stages of payment out of TotalsView, into a new view. By doing that, we can have separate, simpler handling for reader disconnection on these screens.

**Comment by dangermattic** [comment]
Thanks for reporting! 👍

**Comment by malinajirka** [comment]
@joshheald Is this issue still valid? If it is, please assign a priority to it and move it to our backlog, thanks!

**Comment by joshheald** [comment]
Not really; the first example shows a sensible error saying the payment couldn't be completed because there's no card reader connected. 

The second and third don't happen anyway, I think because of the card reader status not updating when bluetooth is turned off, as [noted](https://github.com/woocommerce/woocommerce-ios/issues/13738#issuecomment-2615273012) in #13738

I'll make an issue to investigate the lack of updates to the card reader status from external events; if we fix that, I'd want to re-test these.

**Comment by malinajirka** [comment]
Thanks @joshheald, since it seems you plan to follow up on this, could you please assign a priority and move it to our backlog?

P.S. It seems it's marked as `Task` even though it sounds like a `Bug`, consider recategorizing it, thanks!

**Comment by joshheald** [comment]
After fixing the the issue where `Reader connected` didn't update to `Connect your reader` after turning off bluetooth, this becomes reproducible again.

I can't make it do anything bad, it's just visual, so I'm leaving this as low priority. When I fix #13738 it may improve this as well.
